scripts/build-xcframework.sh

```diff
@@ -66,3 +67,5 @@ generate_xcframework "Sentry" "-Dynamic"
 generate_xcframework "Sentry" "" staticlib
 
 generate_xcframework "SentrySwiftUI"
+
+generate_xcframework "Sentry" "-NoUIKit" mh_dylib Release_without_UIKit
```

Comment:
`m`: I think that's a bit cleaner
```suggestion
generate_xcframework "Sentry" "-WihoutUIKitOrAppKit" mh_dylib Release_without_UIKitOrAppKit
```

Reply:
The forth parameter is the configuration name. It cannot be changed

Reply:
I thought that's how the XCFramework is called. I only care about the name of the XCFramework. How is that one called then?

Reply:
Second parameter.
It will be `Sentry-WihoutUIKitOrAppKit`